bundles/org.openhab.binding.chatgpt/src/main/java/org/openhab/binding/chatgpt/internal/ChatGPTHandler.java

```diff
@@ -31,7 +33,6 @@
 import org.openhab.binding.chatgpt.internal.dto.ChatResponse;
 import org.openhab.core.io.net.http.HttpClientFactory;
 import org.openhab.core.library.types.StringType;
-import org.openhab.core.thing.Channel;
```

Comment:
Why should **org.openhab.core.thing.Channel** be swapped for **java.nio.channels.Channel**?

Reply:
VS Code did some nasty import stuff when editing files outside of the current folder. Fixed all of them.